flow-report/test/header-test.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import fs from 'fs';
+import {dirname} from 'path';
+import {fileURLToPath} from 'url';
+
+import {FunctionComponent} from 'preact';
+import {render} from '@testing-library/preact';
+
+import {Header} from '../src/header';
+import {FlowResultContext} from '../src/util';
+
+const flowResult = JSON.parse(
+  fs.readFileSync(
+    // eslint-disable-next-line max-len
+    `${dirname(fileURLToPath(import.meta.url))}/../../lighthouse-core/test/fixtures/fraggle-rock/reports/sample-lhrs.json`,
```

Comment:
LH_ROOT :)

Reply:
I don't think this is possible without making a reference to the root `tsconfig.json`. I could make a separate LH_ROOT in this directory or move this to it's own file. WDYT?